spec/controllers/projects_controller_spec.rb

```diff
@@ -254,6 +254,43 @@
       before{ get :show, permalink: project.permalink, project_post_id: project_post.id, locale: :pt }
       it("should assign update to @update"){ expect(assigns(:post)).to eq(project_post) }
     end
+
+    context "@supporters" do
+      let(:project) { create(:project) }
+
+      before do
+        allow_any_instance_of(Project).to receive(:recurring?).and_return(true)
+      end
+
+      context "when there are paid subscriptions for the project" do
+        before do
+          user_1 = create(:user, name: 'fulano')
```

Comment:
By mapping a resource's identifier is easier to understand what's the problem, when it happens.

If you assert using the hole object, when it fails, the message is something like:
```Ruby
Failure/Error: expect(assigns(:supporters).map(&:name)).to contain_exactly(fulano)
       expected collection contained:  [#<User id: 5, email: "person5@example.com", name: "Foo bar", bio: "This is Foo bar's biography.", image_url: nil, newsletter: false, project_updates: false, created_at: "2017-04-10 19:38:54", updated_at: "2017-04-10 19:38:54", admin: false, full_name: nil, address_street: nil, address_number: nil, address_complement: nil, address_neighbourhood: nil, address_city: nil, address_state: nil, address_zip_code: nil, phone_number: nil, locale: "pt", cpf: nil, encrypted_password: "$2a$04$iCPwcLUKej.PFBYRmvB5SuynzD4COeH/3MR7pjdjIVQ...", reset_password_token: nil, reset_password_sent_at: nil, remember_created_at: nil, sign_in_count: 0, current_sign_in_at: nil, last_sign_in_at: nil, current_sign_in_ip: nil, last_sign_in_ip: nil, twitter: nil, facebook_link: nil, other_link: nil, uploaded_image: nil, moip_login: nil, state_inscription: nil, channel_id: nil, deactivated_at: nil, reactivate_token: nil, address_country: nil, country_id: nil, access_type: 0, birth_date: nil, responsible_name: nil, gender: nil, mobile_phone: nil, responsible_cpf: nil, doc1: nil, doc2: nil, doc3: nil, doc4: nil, doc5: nil, doc6: nil, doc7: nil, doc8: nil, doc9: nil, doc10: nil, doc11: nil, doc12: nil, doc13: nil, doc_updated_at: nil, approved_at: nil, staff: nil, job_title: nil, uuid: nil, uncrypted_password: nil, original_doc1_url: nil, original_doc2_url: nil, original_doc3_url: nil, original_doc4_url: nil, original_doc5_url: nil, original_doc6_url: nil, original_doc7_url: nil, original_doc8_url: nil, original_doc9_url: nil, original_doc10_url: nil, original_doc11_url: nil, original_doc12_url: nil, original_doc13_url: nil, staffs: []>]
       actual collection contained:    []
```

But, if you use `map` on an object identifier, we can have the following message:
```Ruby
Failure/Error: expect(assigns(:supporters).map(&:name)).to contain_exactly('fulano', 'beltrano')
       expected collection contained:  ["beltrano", "fulano"]
       actual collection contained:    []
```
It looks too much easier to read and understand what's the problem.